tests/test_service.py

```diff
@@ -285,7 +257,6 @@ def test_positive_fm_service_restart_bz_1696862(setup_bz_1696862, ansible_module
         assert result["rc"] == 0
 
 
-@pytest.mark.capsule
 def test_positive_foreman_maintain_service_list_sidekiq(ansible_module):
```

Comment:
@Gauravtalreja1 why we are removing this capsule test?

Reply:
Updated reason in the description.